### PR TITLE
NOISSUE - Change endpoint /version by /health

### PR DESCRIPTION
**Pull request by manuio**
Signed-off-by: Manuel Imperiale <manuel.imperiale@gmail.com>

### CODE REVIEW

**Review by blokovi — DISMISSED** [review]

File: docs/edge.md [hunk 1]
```diff
@@ -295,7 +295,7 @@ Values from environment variables will be used to populate export.toml
 - `port` - HTTP port where status of `Export` service can be fetched.
 ```bash
 curl -X GET http://localhost:8170/health
-{"description":"Mainflux export service","version":"0.0.1", "status": "pass"}%
+'{"status": "pass", "version":"0.12.1", "commit":"57cca9677721025da055c47957fc3e869e0325aa" , "description":"export service", "build_time": "2022-01-19_10:13:17"}'
```
Comment (drasko) [inline docs/edge.md:298]: Looing at [here](https://datatracker.ietf.org/doc/html/draft-inadarei-api-health-check-06#section-5), I do not see `"commit"` keyword in the standard. I think you must find and select something closest but tandardized - maybe `releaseId`.

**Review by drasko — CHANGES_REQUESTED** [review]

File: docs/edge.md (hunk 1 above)
Comment (manuio) [reply docs/edge.md:298]: I had the same thought but what about `build_time`? Or we decide to define all in the description or we decide to have custom fields.

Comment (drasko) [reply docs/edge.md:298]: Resolved as per https://github.com/mainflux/mainflux/pull/1541#issuecomment-1020512332

**Review by drasko — APPROVED** [review]
LGTM

**Pull request merged**